release-scripts/tests/test_helm_artifacts.py

```diff
@@ -0,0 +1,31 @@
+import os
+import yaml
+import requests
+
+#
+# Note: this test is only valid for versions >= v3.18.0
+#
+DOCS_PATH = (
+    "/docs"
+    if os.environ.get("CALICO_DOCS_PATH") is None
+    else os.environ.get("CALICO_DOCS_PATH")
+)
+
+with open("%s/_data/versions.yml" % DOCS_PATH) as f:
+    versions = yaml.safe_load(f)
+    RELEASE_VERSION = versions[0]["title"]
+    print("[INFO] using _data/versions.yaml, discovered version: %s" % RELEASE_VERSION)
+
+# Helm version is the calico/node version.
+version = versions[0]["components"]["calico/node"]["version"]
+chart_version = versions[0]["chart"]["version"]
+print("[INFO] using calico/node version for Helm artifacts: %s" % version)
+print("[INFO] using chart version for Helm artifact: %s" % chart_version)
```

Comment:
Seems like this functionality is pretty similar to the `$(YAML_CMD) read - '[0].chart.version'` done in the Makefile. Should we just pass it in as an arg instead?

Reply:
I guess we could, but that same argument applies to basically all the other vars in `versions.yml`.  This is at least consistent with the other post-release tests.